app/views/comments/cud.js.erb

```diff
@@ -0,0 +1,6 @@
+$("#comments-<%= @reference.id %>").html("<%= escape_javascript(render 'papers/comments', reference: @reference) %>");
+$("li#comments-tab-<%= @reference.id %>").html("<%= escape_javascript(render 'papers/comments_tab', reference: @reference) %>");
+$('.toggle-reply').on('click', function(e) {
+  e.preventDefault();
+  $(e.target).parents("ul").next('.comment-reply').toggleClass('hidden');
```

Comment:
I agree.  You mean specifically this function or rewriting this in this render.  I'm not 100% sure why me rendering papers/comment_tab again stops the reply button from working, maybe because the js we have in comments.js only starts after the DOM has loaded?